libs/application/templates/parental-leave/src/lib/directorateOfLabour.utils.ts

```diff
@@ -16,7 +16,8 @@ export const DAYS_IN_MONTH = 30
  * Convert a number of days into a floored number of months
  */
 export const daysToMonths = (days: number) => {
-  const fullMonths = Math.floor(days / DAYS_IN_MONTH)
+  const ceilOrFloor = days < 0 ? Math.ceil : Math.floor
```

Comment:
I wouldn't touch this function directly and handle this case in the SummaryRights component instead where the function is used: `const given = daysToMonths(giveDays)`.

Reply:
Sounds good, just updated it now. 
I've set it to `const given = daysToMonths(Math.abs(giveDays))`. 
It's using `Math.abs` because `daysToMonths` will floor in the wrong direction that we want:
`daysToMonths(-45) === -2.5` when it should be `1.5`